### DISCUSSION TITLE
Arbitrary background position (x and y axis)

### DISCUSSION THREAD

**Discussion by swk23C8** [tailwindlabs/tailwindcss#8518] [Help] [answered]
I've read the discussion #4626 and issue #7997 prior to starting the discussion.

I don't think using arbitrary values for background position (x and y)  is implemented yet in the latest version of tailwindcss
I'm currently using a hacky way as suggested in #7997

e.g.
`bg-[left_calc(-200%)_top_calc(0%)]`
instead of doing something like
`bg-[-200% 0%]`

to achieve the same result as doing
`background-position: -200% 0%;`
or
```
background-position-x: -200%;
background-position-y: 0%
```

 I was wondering if there is any downside of doing it in such manner?

**Comment by adamwathan** [comment, marked as answer]
Hey! Should be able to just hint the type and it'll work:

<img width="667" alt="image" src="https://user-images.githubusercontent.com/4323180/172009053-87a8d946-db6a-4df4-a362-9d664f6c283e.png">

**Reply by swk23C8** [reply]
thank you! that's perfect

**Comment by ollyde** [comment]
For anyone else who doesn't want to manually type it from an image..

`bg-[position:-200%_0%]`
